README.md

```diff
@@ -55,25 +41,42 @@ user_training_id user_id training_id training_date
 ```
 Write a query to get the list of users who took a training lesson more than once in the same day, grouped by user and training lesson, each ordered from the most recent lesson date to oldest date.
 
-## Task 5 - Linux
+## Task 4 - Linux
 How would you:
 - SSH into a server with IP 12.34.56.78, user "ec2-user" and identity file "MyPrivateKey.key"
 - Go to folder "/var/logs"
 - Show on console the latest 30 lines of the file "super-app.log"
 
-## Task 6 - Automated end-to-end web testing
-With a test automation framework of your choice, test [Wikipedia](https://wikipedia.org)'s search feature by searching for "Voyager 1". We've already evaluated your test case generation skills on a previous task, so there's no need to implement more than just the specified user flow.
+## Task 5 - Automated end-to-end web testing
+With a test automation framework of your choice, test [Wikipedia](https://wikipedia.org)'s search feature by searching for "Voyager 1".
+
+You can use whatever language you desire (we use Js + Mocha + Cypress/Selenium).
 
-You can use whatever language you desire (we use Js + Mocha + Cypress/Selenium). Use as many features as you want of the testing framework, use as many patterns you want to. The required reporting is console output, but having a nicer reporting is a plus. The cleaner the code, the better.
+As an alternative, if you feel more comfortable, you can test the mobile app of Wikipedia on Android or iOS, either is fine (you can test both if you like). Once again, no constraint on the choice of the framework (we use Js + Mocha + WebdriverIO with Appium).
 
-We’ll evaluate your programming skills and web automation skills.
+Use as many features of the testing framework as you want, use as many design patterns you want. The required reporting is console output but having nicer reporting is a plus. The cleaner the code, the better.
+
+We will evaluate your programming skills and web automation skills.
+
+## Task 6 - API testing & test case generation
+Given the REST API:
+https://developers.giphy.com/docs/api/endpoint#search
+
+And this call to retrieve a list of GIFs:
+http://api.giphy.com/v1/gifs/search?q=funny+dog&api_key=dc6zaTOxFJmzC
+
+Your third task is to extensively test that API endpoint. You can use the provided *api_key*: it’s a public key for testing out that API. Please, read the documentation before starting.
```

Comment:
```suggestion
Your task is to extensively test that API endpoint. You can use the provided *api_key*: it’s a public key for testing out that API. Please, read the documentation before starting.
```

Reply:
leftover from the old version, thank you